tests/logictest/suites/gen/06_show/06_0003_show_settings_v2

```diff
@@ -34,6 +34,7 @@ SHOW SETTINGS LIKE 'enable%';
 
 ----
 enable_async_insert 0 0 SESSION Whether the client open async insert mode, default value: 0 UInt64
+enable_bloom_filter_index 0 0 SESSION Enable bloom filter index (if applicable for the underlying table engine) by setting this variable to 1, default value: 0	UInt64
```

Comment:
a little vague: set it to  0 affect insert and/or select(suppose it was inserted with bloom filter ) ?

Reply:
The original idea is 
- only if `enable_bloom_filter_index` is enabled, during insertion/mutation, index will be built.
- if it is disabled, even if the index exists, it will not be used. Mainly for quick toggling of the index feature during performance evaluation.

But as @BohuTANG suggested, this setting flag is removed. bloom index is always on for now :)